setup.cfg

```diff
@@ -0,0 +1,31 @@
+[pycodestyle]
+max-line-length = 99
+
+[pylint.MASTER]
+jobs = 3
+max-line-length = 99
+fail-under=9.9
+
+load-plugins =
+  pylint.extensions.bad_builtin,
+  pylint.extensions.broad_try_clause,
+  pylint.extensions.check_elif,
+  pylint.extensions.comparetozero,
+  pylint.extensions.emptystring,
+  pylint.extensions.overlapping_exceptions
```

Comment:
Is there any strong reason why you skipped `pylint.extensions.empty_comment`? 🤔  I hink, we don't want to have empty comments in the code, do we?